applications/protocols_serialization/server/src/main.c

```diff
@@ -30,5 +38,11 @@ int main(void)
 
 	LOG_INF("RPC server ready");
 
+	ret = dk_buttons_init(button_handler);
```

Comment:
This is the same. We shouldn't pollute source by stub testing.

Reply:
This meant to be a way to test this feature in our CI, not stub testing.

Reply:
Mark as solved to not duplicate discussion started above.